models/us_census_acs/code/clean_profiles.py

```
@@ -0,0 +1,221 @@
+        outfile = f"{outdir}/{plabel}.parquet"
+        if os.path.exists(outfile):
+            print(f"  SKIP {level} {plabel} {y}")
+            continue
+        tmp = outfile + ".tmp"
+        writer = None  # created lazily on first non-empty batch
+        total = 0
+        for fp in files:  # one row-group per source file -> bounded memory
+            rows = []
+            try:
+                melt_file(fp, level, y, plabel, rows)
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)